{{ cookiecutter.project_slug }}/tf/provider.tf

```diff
@@ -11,14 +12,12 @@ terraform {
 }
 
 provider "aws" {
-  region  = local.region
-  profile = local.profile
+  region  =  "ap-southeast-2"
+  profile = var.CI ? "" : var.profile
 
   default_tags {
     tags = {
-      # NB: Capitalization is correct and required
-      Project = local.project
```

Comment:
Did you find the project tag not useful ?

Reply:
It was essentially a duplicate of repo, although I did add another option in the config to differentiate, don't really have an opinion on this